boards/others/candlelightfd/candlelightfd.dtsi

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2024 Sean Nyekjaer <sean@geanix.com>
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g0/stm32g0b1Xb.dtsi>
+#include <st/g0/stm32g0b1r(b-c-e)tx-pinctrl.dtsi>
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,canbus = &fdcan1;
+	};
+
+	aliases {
+		led0 = &led_rx;
+		led1 = &led_tx;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_rx: led_rx {
+			gpios = <&gpioa 3 GPIO_ACTIVE_LOW>;
+			label = "LED RX";
+		};
+		led_tx: led_tx {
+			gpios = <&gpioa 4 GPIO_ACTIVE_LOW>;
+			label = "LED TX";
+		};
+	};
+
+	transceiver0: can-phy0 {
+		compatible = "nxp,tja1051", "can-transceiver-gpio";
+		enable-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+		max-bitrate = <5000000>;
+		#phy-cells = <0>;
+	};
+
+	transceiver1: can-phy1 {
+		compatible = "nxp,tja1051", "can-transceiver-gpio";
+		enable-gpios = <&gpiob 2 GPIO_ACTIVE_LOW>;
+		max-bitrate = <5000000>;
+		#phy-cells = <0>;
+	};
```

Comment:
This needs to be disabled by default (`status = "disabled"`), but why not move it - along with the `fdcan2` node - to `candlelightfd_stm32g0b1xx_dual.dts` as they are only relevant for that board variant?